package.json

```diff
@@ -45,12 +45,22 @@
     "eslint-plugin-node": "^8.0.0",
     "eslint-plugin-promise": "^4.0.0",
     "eslint-plugin-standard": "^4.0.0",
-    "istanbul": "^0.4.5",
     "jasmine": "^3.4.0",
     "jasmine-spec-reporter": "^4.2.1",
+    "nyc": "^14.1.1",
     "osenv": "^0.1.3",
     "promise-matchers": "^0.9.6",
     "rewire": "^4.0.1"
   },
-  "contributors": []
+  "contributors": [],
+  "nyc": {
+    "include": [
+      "cordova-common.js",
+      "src/**"
+    ],
```

Comment:
I would favor an _everything-but_ approach as I did in [cordova-eslint](https://github.com/apache/cordova-eslint/blob/1ec460f7c223edcfa5e62a7f69443f8f0964d76d/package.json#L36-L41). This would have two advantages:
- It includes files that have 0% coverage in the report
- You notice when the patterns need to be updated, because files show up in coverage reports that shouldn't be there

Reply:
I will change to `"all": true,`